apps/api/scripts/competition-status.ts

```diff
@@ -232,14 +244,15 @@ async function showCompetitionStatus() {
           );
 
           if (sortedSnapshots.length > 0) {
-            initialSnapshots.set(entry.teamId, sortedSnapshots[0].totalValue);
+            initialSnapshots.set(entry.teamId, sortedSnapshots[0]!.totalValue);
```

Comment:
We can force this because we know there is at least one item in the array because of the check above.

Reply:
makes sense!